hera_sim/visibilities/vis_cpu.py

```diff
@@ -43,17 +53,24 @@ def __init__(self, bm_pix=100, precision=1, use_gpu=False, **kwargs):
             self._complex_dtype = np.complex128
 
         if use_gpu:
+            if use_pixel_beams:
+                raise RuntimeError("GPU cannot be used with pixel beams") 
```

Comment:
I think @AaronParsons might have an issue with this. The original idea for `vis_cpu` was that it should be a drop-in replacement for `vis_gpu`. This breaks the symmetry. I'll let him have a say here.